.pre-commit-config.yaml

```diff
@@ -14,13 +14,24 @@ repos:
   - id: debug-statements
   - id: forbid-new-submodules
 - repo: https://github.com/psf/black
-  rev: stable
+  rev: 19.10b0
   hooks:
   - id: black
 - repo: https://gitlab.com/pycqa/flake8
-  rev: 3.8.1
+  rev: "3.8.1"
   hooks:
   - id: flake8
+    additional_dependencies: ["flake8-bugbear == 20.1.2"]
+    language_version: python3.8
+- repo: https://github.com/asottile/seed-isort-config
```

Comment:
instead manually populating known_third_party in setup.cfg use this tool to autogenerate

Reply:
Is that what added setuptools? That seems pointless, as it's only used in `setup.py`, but I guess it's harmless.

Reply:
Well, the list here is for all python files in the projects. And setup.py is also a python file within this project. E.g. if you would have a Sphinx ``conf.py`` that pulls in sphinx modules via import that would also show up here. But it's harmless as you said.